tooling/scripts/download-publisher-cli.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"bytes"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/JetBrains/qodana-cli/v2025/platform/utils"
+)
+
+func main() {
+	if len(os.Args) <= 1 {
+		log.Fatalf("Expected publisher-cli version as first argument")
+	}
+	version := os.Args[1]
+
+	url := fmt.Sprintf("https://packages.jetbrains.team/maven/p/ij/intellij-dependencies/org/jetbrains/qodana/publisher-cli/%s/publisher-cli-%s.jar", version, version)
+	utils.DownloadFile("publisher-cli.jar", url, nil)
+
+	actualSha256Bytes, err := utils.GetFileSha256("publisher-cli.jar")
+	if err != nil {
+		log.Fatalf("Error while calculating SHA-256: %s", err)
+	}
+	actualSha256 := hex.EncodeToString(actualSha256Bytes[:])
+
+	expectedSha256Url := url + ".sha256"
```

Comment:
How big is the value of SHA256 verification if both artifacts (sha256 and .jar) are served on the same host? I prefer the hash embedded into project's source code, as it's more secure

Reply:
And guarantees the reproducibility

Reply:
Fixed